clientd3d/lagbox.c

```diff
@@ -284,30 +297,23 @@ LRESULT Lagbox_OnTooltipCallback(HWND hWnd, TOOLTIPTEXT* pTTT)
 //
 void Lagbox_Command(HWND hWnd, int id, HWND hwndCtrl, UINT uNotify)
 {
-	object_node lagbox;
-	TOOLTIPTEXT ttt;
-
 	// Make a temporary copy of the lagbox to view close up.
 	// This allows it to spin freely and not affect the real lagbox.
-	// Unfortunately, this means it won't change colors and speed in
-	// the close-up window.
-	//
+
 	if (!pobjLagbox)
 		return;
-	lagbox = *pobjLagbox;
-	lagbox.animate = &lagbox.normal_animate;
-	lagbox.overlays = &lagbox.normal_overlays;
-	lagbox.flags |= OF_PLAYER | PF_OUTLAW;
 
-	Lagbox_OnTooltipCallback(hWnd, &ttt);
+	latencyDialogActive = true;
 
-	SetDescParams(hWnd, DESC_NOAGE);
-	DisplayDescription(&lagbox,
+	SetDescParams(hWnd, !DESC_DROP);
```

Comment:
This looks suspicious.  SetDescParams takes an int, but you're passing in false (!DESC_DROP).